### PR TITLE
Adds UsersController base and only the GET method

### PR DESCRIPTION
**Pull request by cristianbgp**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    render json: User.all
```
Comment (liannivin) [inline app/controllers/users_controller.rb]: Good work!

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      User.delete_all
+    end
+
+    it 'returns http status ok' do
+      get :index
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'render json with all products' do
+      User.create(fullname: 'Cristian Granda', username: 'cristianbgp', profile_picture: 'https://loremflickr.com/320/240', bio: 'something', website: 'https://cristianbgp.com')
```
Comment (liannivin) [inline spec/controllers/users_controller_spec.rb]: I like it!

File: spec/models/like_spec.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # pending "add some examples to (or delete) #{__FILE__}"
```
Comment (yummta) [inline spec/models/like_spec.rb]: Thanks for that

File: spec/controllers/users_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe UsersController do
+  describe 'GET index' do
+    
+    before do
+      User.delete_all
+    end
```
Comment (yummta) [inline spec/controllers/users_controller_spec.rb]: Why do you need this?

Comment (cristianbgp) [reply spec/controllers/users_controller_spec.rb]: Because we will need this with the other http methods

**Pull request merged**